src/chat/ChatMessageList.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import ChatMessage from './ChatMessage'
+
+export default function ChatMessageList({chatMessages}) {
```

Comment:
Maybe add a comment about what this function does? Better for someone other than Adam to maintain in the future